### ISSUE TITLE
How can the pdf output be customized further 

### ISSUE THREAD

**Issue by c-myers1** [endroid/qr-code#375]
Thanks for this great library!

I tried to create a multi-line label in 3 ways and they all failed: (1) calling the method twice (2) using \n in the string literal and (3) using the <br>.

Besides, I might want a footer and header.

I know it’s using fpdf but it’s not clear to me how I can achieve such custom output in my code (which uses the builder method.) Clear steps would be appreciated please.

**Comment by endroid** [comment]
Hi @c-myers1 unfortunately this is not supported at the moment.

**Comment by c-myers1** [comment]
Can you give me some bullet points as to how this library can be extended to handle such functionality? What parts need to be modified and what do I need to be aware of?

**Comment by endroid** [comment]
Hi @c-myers1 you can create your own writer based on the [PNG writer](https://github.com/endroid/qr-code/blob/master/src/Writer/PngWriter.php). Loading the matrix and generatoring the QR code remains the same but you can add an extra step within the write method for calculating and placing the image. Good luck!

**Comment by endroid** [comment]
Of course. You can create your own custom writer based on the [PDF writer](https://github.com/endroid/qr-code/blob/master/src/Writer/PdfWriter.php) and the label writing logic is contained within the "if ($label instanceof LabelInterface) ..." blocks so you can make your modifications there. Good luck!
